seleniumByTechnocredits/src/seleniumPractise/purva/Assignment1_6.java

```diff
@@ -0,0 +1,387 @@
+		}
+		Thread.sleep(3000);
+		alert15.accept();
+
+		// input 2 = purvah for password
+		Thread.sleep(3000);
+		driver.findElement(By.cssSelector("input#passwordReg")).clear();
+		Thread.sleep(3000);
+		driver.findElement(By.cssSelector("input#passwordReg")).sendKeys("PurvaH");
+		Thread.sleep(3000);
+		JavascriptExecutor js16 = (JavascriptExecutor) driver;
+		js16.executeScript("arguments[0].scrollIntoView(true)", element);
+		Thread.sleep(3000);
+		element.click();
+		Alert alert16 = driver.switchTo().alert();
+		String actual_text16 = alert16.getText();
+		String expected_text16 = "password length should be greater then 5 characters.";
+		if (actual_text16.equals(expected_text16)) {
+			System.out.println("Alert 16 actual and expected text match");
+		}
+		Thread.sleep(3000);
+		alert16.accept();
+
+		// below code is for re-type password field where retype password field
+		// value = purva does not match with password= purvah
+		Thread.sleep(3000);
+		driver.findElement(By.cssSelector("input#repasswordReg")).sendKeys("Purva");
+		Thread.sleep(3000);
+		JavascriptExecutor js17 = (JavascriptExecutor) driver;
+		js17.executeScript("arguments[0].scrollIntoView(true)", element);
+		Thread.sleep(3000);
+		element.click();
+		Alert alert17 = driver.switchTo().alert();
+		String actual_text17 = alert17.getText();
+		String expected_text17 = "retype password donot match.";
+		if (actual_text17.equals(expected_text17)) {
+			System.out.println("Alert 17 actual and expected text match");
+		}
+		Thread.sleep(3000);
+		alert17.accept();
+
+		// below code is for re-type password field where retype password field
+		// value = purvah matches with password= purvah
+		Thread.sleep(3000);
+		driver.findElement(By.cssSelector("input#repasswordReg")).clear();
+		Thread.sleep(2000);
+		driver.findElement(By.cssSelector("input#repasswordReg")).sendKeys("PurvaH");
+		Thread.sleep(3000);
+		JavascriptExecutor js18 = (JavascriptExecutor) driver;
+		js18.executeScript("arguments[0].scrollIntoView(true)", element);
+		Thread.sleep(3000);
+		element.click();
+		Alert alert18 = driver.switchTo().alert();
+		String actual_text18 = alert18.getText();
+		String expected_text18 = "Please agree to terms of service and privacy policy";
+		if (actual_text18.equals(expected_text18)) {
+			System.out.println("Alert 18 actual and expected text match");
+		}
+		Thread.sleep(3000);
+		alert18.accept();
+
+		// below code is to check agreement checkbox and click on submit button
+		// which generates success alert box
+		Thread.sleep(3000);
+		driver.findElement(By.cssSelector("label.checkbox")).click();
+		Thread.sleep(3000);
+		element.click();
+		Thread.sleep(2000);
+		Alert alert19 = driver.switchTo().alert();
+		String actual_text19 = alert19.getText();
+		String expected_text19 = "Success";
+		if (actual_text19.equals(expected_text19)) {
+			System.out.println("Alert 19 actual and expected text match");
+		}
+		Thread.sleep(3000);
+		alert19.accept();
+	}
+
+}
```

Comment:
You can write expected alert text directly in equal condition. This will reduce number of variables and can be useful in code optimization.
You can add code for closing the browser after completion of validation.
You can write all the code in one different method and just call that method in main as we should avoid writing code in main method.